recipes/gcc/all/conanfile.py

```diff
@@ -0,0 +1,126 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanException, ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.29.1"
+
+
+class GccConan(ConanFile):
+    name = "gcc"
+    description = "The GNU Compiler Collection includes front ends for C, " \
+                  "C++, Objective-C, Fortran, Ada, Go, and D, as well as " \
+                  "libraries for these languages (libstdc++,...). "
+    topics = ("gcc", "gnu", "compiler", "c", "c++")
+    homepage = "https://gcc.gnu.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "GPL-3.0-only"
+    settings = "os", "compiler", "arch", "build_type"
+    _autotools = None
+
+    def build_requirements(self):
+        self.build_requires("flex/2.6.4")
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        pkgversion = 'conan GCC %s' % self.version
+        bugurl = self.url + '/issues'
+        libdir = "%s/lib/gcc/%s" % (self.package_folder, self.version)
+        args = [
+            "--enable-languages=c,c++",
```

Comment:
```suggestion
            "--enable-languages=c,c++,fortran",
```

Because gfortran recipe is broken. It requires more artifacts which are part of gcc.

Reply:
Is this option then building the Fortran compiler executable?

Reply:
Yes. The size is too small to complain. I just built GCC 10 past days and the final result is almost same.

Reply:
But since we already have an own package, it should probably be in there 🤔 At least I don't think I would necessarily expect Fortran in this GCC package. What do others think?

Reply:
Having fortran available, is useful for building `BLAS/LAPACK`-type libraries.
The issue is that fortran is not available on the cci builders, so it won't be possible to build/test it on cci ci.

Reply:
@madebr I don't argue that having Fortran is good. But we have a separate Fortran package already, so I think it should stay there and not be added to the GCC package

Reply:
You're right but there is a slight difference.
gfortran is packaging a binary and this one is built from source.
So this one has potential for more support, such as the bds'es.

Reply:
What if people want to use their own compiler, but a Fortran compiler via this package? Or use GCC via this package but the LLVM Fortran compiler from such a (hopefully) upcoming LLVM package? Am I overthinking this? 😄

Reply:
not impossible, indeed. It could be optional, of course. I can add an option for fortran compiler.